.github/workflows/lint-codeql.yaml

```diff
@@ -3,12 +3,24 @@ name: codeql
 on:
   push:
     branches:
+    - master
     - v1.10
     - v1.9
     - v1.8
+    paths:
+      - '*.go'
+      - 'go.mod'
+      - 'go.sum'
   pull_request:
     branches:
     - master
+    - v1.10
+    - v1.9
+    - v1.8
```

Comment:
Shouldn't each one of these lines be in its own branch? :thinking:

Reply:
I don't understand what you're asking.

Reply:
We have a `lint-codeql.yaml` specific for each branch so the `pull_request` should be pointing only to `master`, in the v1.10 branch it should only refere to `v1.10` and so on. Does it make sense?

Reply:
it's probably easier to just do:
```
pull_request: {}
```